defaults/main.yml

```diff
@@ -97,83 +97,81 @@ mattermost_ClientRequirements_IosLatestVersion: ""
 mattermost_ClientRequirements_IosMinVersion: ""
 
 mattermost_Sql_DriverName: "postgres"
-#Sql_DataSource: "postgres://{{ db_user }}:{{ db_password }}@127.0.0.1:5432/{{ db_name }}?sslmode=disable&connect_timeout=10"
-#Sql_DataSource is implemented with the template mattermost-config.json.j2 - setting it here may have strange consequences
 mattermost_Sql_DataSourceReplicas: []
 mattermost_Sql_DataSourceSearchReplicas: []
 mattermost_Sql_MaxIdleConns: 20
 mattermost_Sql_MaxOpenConns: 300
-mattermost_Sql_Trace: "false"
+mattermost_Sql_Trace: false
 mattermost_Sql_AtRestEncryptKey: "jksksj85pzbomhs3ockhrzynpgh8ox34"
 mattermost_Sql_QueryTimeout: 30
 
-mattermost_Log_EnableConsole: "true"
+mattermost_Log_EnableConsole: true
 mattermost_Log_ConsoleLevel: "INFO"
-mattermost_Log_EnableFile: "true"
+mattermost_Log_EnableFile: true
 mattermost_Log_FileLevel: "INFO"
 mattermost_Log_FileFormat: ""
 mattermost_Log_FileLocation: ""
-mattermost_Log_EnableWebhookDebugging: "true"
-mattermost_Log_EnableDiagnostics: "true"
+mattermost_Log_EnableWebhookDebugging: true
+mattermost_Log_EnableDiagnostics: true
 
 mattermost_Password_MinimumLength: 5
-mattermost_Password_Lowercase: "false"
-mattermost_Password_Number: "false"
-mattermost_Password_Uppercase: "false"
-mattermost_Password_Symbol: "false"
+mattermost_Password_Lowercase: false
+mattermost_Password_Number: false
+mattermost_Password_Uppercase: false
+mattermost_Password_Symbol: false
 
-mattermost_File_EnableFileAttachments: "true"
-mattermost_File_EnableMobileUpload: "true"
-mattermost_File_EnableMobileDownload: "true"
+mattermost_File_EnableFileAttachments: true
+mattermost_File_EnableMobileUpload: true
+mattermost_File_EnableMobileDownload: true
 mattermost_File_MaxFileSize: 52428800
 mattermost_File_DriverName: "local"
 mattermost_File_Directory: "./data/"
-mattermost_File_EnablePublicLink: "false"
+mattermost_File_EnablePublicLink: false
 mattermost_File_PublicLinkSalt: "ey3hcypcwwyqun3nok4dx5k3iiaepqda"
```

Comment:
what about this salt? Is it okay for it to be public?

Reply:
fixed